sunkit_pyvista/plotter.py

```diff
@@ -72,6 +74,25 @@ def set_camera_coordinate(self, coord):
         pos = tuple(camera_position[0])
         self.plotter.camera.position = pos
 
+    def set_view_angle(self, angle: u.deg = 30):
+        """
+        Sets the view angle of the camera by the specified value
+        in degrees.
+
+        Parameters
+        ----------
+        angle : `astropy.units.Quantity`
+            The viewing angle.
+        """
+        view_angle = angle.to_value()
```

Comment:
```suggestion
        view_angle = angle.to_value(u.deg)
```

Reply:
if our `view_angle` is specified in other units as well, won't we have to account for that here? Because self.camera.view_angle is in degrees?

Reply:
Angle can be in different units, but units that are still convertible to degrees (e.g. radians), so this call forces the value to be the value in degrees, which is what pyvista wants.

Reply:
Oh my bad, I thought you removed `to_value(u.deg)` :sweat_smile:, I get it now, my bad, I had made this change locally, I forgot to commit